src/noop.jl

```diff
@@ -53,16 +53,18 @@ Note that this method may return a wrong position when
 - some data have been inserted by `TranscodingStreams.unread`, or
 - the position of the wrapped stream has been changed outside of this package.
 """
-function Base.position(stream::NoopStream)
+function Base.position(stream::NoopStream)::Int64
```

Comment:
I'm trying to decide if this should be `Int64` or `Int`. The latter varies based on the system word size.

```
$ julia +1.10~x86 -E "typeof(0)"
Int32
```

Reply:
Cool, I didn't know juliaup had 32-bit julia.

This should be `Int64` to support large files.